backend/src/main/java/com/huellapositiva/domain/service/VolunteerService.java

```diff
@@ -28,7 +30,8 @@ public Integer registerVolunteer(PlainPassword plainPassword, EmailConfirmation
             ExpressRegistrationVolunteer expressVolunteer = new ExpressRegistrationVolunteer(hash, emailConfirmation);
             return volunteerRepository.save(expressVolunteer);
         } catch (DataIntegrityViolationException ex) {
-            throw new UserAlreadyExists();
+            log.error("Unable to persist volunteer due to a conflict.", ex);
```

Comment:
If we do not log the error we wont be able to identify issues at the time of creating a user.